static/js/buttons/profiles/DeleteProfileBtn.tsx

```diff
@@ -8,9 +8,20 @@ import { deleteProfile } from "../../api/profiles";
 interface Props {
   name: string;
   notify: NotificationHelper;
+  appearance?: string;
+  className?: string;
+  isDense?: boolean;
+  label?: string;
 }
 
-const DeleteProfileBtn: FC<Props> = ({ name, notify }) => {
+const DeleteProfileBtn: FC<Props> = ({
+  name,
+  notify,
+  appearance = "base",
+  className = "p-contextual-menu__link",
```

Comment:
I think the button props can be unified and simplified. In the current situation handling the possible customisations and settings is getting a bit too noisy.

How about a single prop on all *Btn classes with name btnStyle. It is an object with all props from Button component in react-conponents and typed accordingly. In the *Btn component we just spread that object into the Button. Hence all our *Btn can be fully customized in the use case.

Also we can have some presets for the btnStyle that can be imported from any use case. So we don't repeat the btnProps object in our components, but just import it.

Does it make sense?

Reply:
Sure, nice idea. I can submit a dedicated PR to implement this over all the *Btn components.
But yeah, I also agree that it is out of scope for this PR. So I will mark it as solved even though I have not applied any change to the code in this regard.